wordpress-mobile/WordPress-Android
# Add theme activation action sheet

Part of #15014

Fixes #
This PR adds an action sheet that enables the user to keep their homepage layout when activating a theme. This action sheet is not linked to any control in this PR right now.

To test:

1. Open the app
2. Scroll down and select Themes under the Look and Feel
3. Scroll to your desired theme
4. Press the three dots(…) button
5. Press Activate
6. Verify that the new dialog appears (screenshot in comment)

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
